daemon/src/connection.rs

```diff
@@ -483,10 +491,17 @@ impl Actor {
     }
 
     fn handle_measure_pulse(&mut self, _: MeasurePulse) {
+        tracing::trace!("measuring pulse");
+
```

Comment:
🔧 I guess you didn't want to commit this.

Reply:
thanks for doublechecking, but I wanted to commit this - it's only at trace level, so no impact on production :)

Reply:
(I have added it as a `wire` target though, so it's a bit more explicit - I need it for debugging)

Reply:
Fair enough! I assumed it was temporary because its all lowercase :P

Reply:
I can see that! this is probably good heuristic! I try to remind myself with temporary `FIXME` or `TODO` comments if I add something temporarily, as magit-todos tells me I have some new ones introduced on my branch!